meson.build

```diff
@@ -39,18 +39,24 @@ sqlite3_dep = dependency(
 sqlitecpp_incl = [ 
     include_directories('include')
 ]
-sqlitecpp_srcs = [
+sqlitecpp_srcs = files(
     'src/Backup.cpp',
     'src/Column.cpp',
     'src/Database.cpp',
     'src/Exception.cpp',
     'src/Savepoint.cpp',
     'src/Statement.cpp',
     'src/Transaction.cpp',
-]
-sqlitecpp_args = [
-    '-Wall',
-]
+)
+sqlitecpp_args = cxx.get_supported_arguments(
+    # included in meson by default
+    # -Wall
+    '-Wextra',
+    '-Wpedantic',

```

Comment:
Wextra and Wpedantic are in meson too, simply set the `default_options: ['warning_level=3']`

Reply:
changed